UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -143,6 +143,12 @@ public static void Start()
 
         public static void ReadyForPlayers()
         {
+            if(CurrentState.CurrentGameState == GameState.Active)
+            {
+                string msg = "Cannot call ReadyForPlayers on an Active server";
+                Debug.Log(msg);
+                throw new Exception(msg);
```

Comment:
Exception will show in the log properly as an error, no need to debug log it and throw it, just throw it.

Reply:
done, thanks!